File: March 4, 2026.js
function cardValues(cards) {

  let arr = [];

  for (let card of cards) {
    if(card.includes("A")) {
      arr.push(1);
    } else if (/[JQK]/.test(card)) {
      arr.push(10);
    } else {
      arr.push(parseInt(card.slice(0, -1)));
    }
  }

  return arr;
}

File: March 4, 2026.py
def card_values(cards):

    arr = []
    for card in cards:
        if ("A" in card):
            arr.append(1)
        elif (any(c in card for c in "JQK")):
            arr.append(10)
        else: 
            arr.append(int(card[:-1]))

    return arr